app/controllers/user_movies_controller.rb

```diff
@@ -0,0 +1,22 @@
+class UserMoviesController < ApplicationController
+  def create
+    @user_movie = UserMovie.new(user_movie_params)
+    @user_movie.save!
+    redirect_to(:back)
+  end
+  
+  def update
+    if @user_movie = UserMovie.find_by(user_id: user_movie_params[:user_id], movie_id: user_movie_params[:movie_id])
+      @user_movie.update_attributes(user_movie_params)
+      redirect_to(:back)
+    else
+      create
```

Comment:
No. It's calling the create function in this controller, not a function on a model.